clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -54,7 +54,7 @@
  * The consumer configuration keys
  */
 public class ConsumerConfig extends AbstractConfig {
-    private static final ConfigDef CONFIG;
+    protected static final ConfigDef CONFIG;
```

Comment:
Why this change? Also what is all this `ReverseXxx` stuff below? Did you add something that does not belong to this PR by accident?

Reply:
Hmm looks like my reverse consumer work got committed by accident. will clean it and go through your comments